app/src/androidTest/java/org/fossasia/openevent/APITest.java

```diff
@@ -83,19 +91,21 @@ public void testSessionAPIResponse() throws Exception {
         APIClient client = new APIClient();
         final CountDownLatch latch = new CountDownLatch(1);
         client.getOpenEventAPI().getSessions(Urls.EVENT_ID, new Callback<SessionResponseList>() {
-            @Override
-            public void success(SessionResponseList sessionResponseList, Response response) {
-                assertNotNull(sessionResponseList.sessions);
-                assertTrue(sessionResponseList.sessions.size() > 0);
-                latch.countDown();
-            }
-
-            @Override
-            public void failure(RetrofitError error) {
-                fail("API Request Failed");
-                latch.countDown();
-            }
-        });
+                    @Override
+                    public void success(SessionResponseList sessionResponseList, Response response) {
```

Comment:
The code is the same and plus your version is grossly over-indented for some reason.